pkgs/servers/krill/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, stdenv
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "krill";
+  version = "0.9.4";
+
+  src = fetchFromGitHub {
+    owner = "NLnetLabs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256:1w5s4vmr126k5lnxsv3hfxx1yz2c1gp3hmwfz5jn4lar78r5kidy";
+  };
+
+  cargoSha256 = "sha256:0ksh68giq3yd8k7vynxdl13ccysl6i5ag2nz9nknbnrvpv201c88";
+
+ buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ Security ];
+ nativeBuildInputs = [ pkg-config ];
+
+  buildNoDefaultFeatures = true;
+
+  # Needed to get openssl-sys to use pkgconfig.
+  OPENSSL_NO_VENDOR = 1; 
+
+  meta = with lib; {
+    description = "RPKI Certificate Authority and Publication Server written in Rust";
+    longDescription = ''
+      Krill is a free, open source RPKI Certificate Authority that lets you run
+      delegated RPKI under one or multiple Regional Internet Registries (RIRs).
+      Through its built-in publication server, Krill can publish Route Origin
+      Authorisations (ROAs) on your own servers or with a third party.
+    '';
+    homepage = "https://github.com/NLnetLabs/${pname}";
+    changelog = "https://github.com/NLnetLabs/${pname}/blob/v${version}/Changelog.md";
```

```suggestion
    homepage = "https://github.com/NLnetLabs/krill";
    changelog = "https://github.com/NLnetLabs/krill/blob/v${version}/Changelog.md";
```
Substituting the name only makes things harder to read imo, and you can't click the links in github.

---

I felt it would make sense be consistent between the homepage and the changelog (which contains ${version}). Is that okay?

---

Version is ok, it makes sense to substitute it because it's less work to update the derivation in case of an update. 

Substituting name only makes things more unreadable, and the homepage link unclickable.

---

Thanks. You are right, I adjusted it.